docs/features/vendor-scanning.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: "Vendor Scanning"
+description: "Understand how CookieChimp scans your website for vendors."
+---
+
+<img src="/images/scan/scan.png" alt="Scan page" />
+
+We scan your website to find the vendors and storage items used on your site.
+We check what storage items were set before and after consent was granted.
+
+If a service runs, that are not supposed to run before consent is granted, we will flag this as `Ran before consent`.
+
+If we detect storage items from a service that is not shown in your banner, we will flag the service as `Not shown in banner`.
+
+If we detect storage items that are not in your banner and we cannot match to a vendor/service, we will flag these as `Not part of a category or service`.
+
+If everything is set up correctly then we flag the service as `All good`.
```

Comment:
We can make all of these be bullet points and the status first and the description after it